aurorawatch-pipeline/test_aurora_load.py

```diff
@@ -0,0 +1,28 @@
+'''Contains tests for the load script'''
+from unittest import mock
+from aurora_load import load_data
+
+
+class TestLoadData:
+    '''Tests for the load_data function'''
+    @mock.patch('aurora_load.get_connection')
+    def test_load_data_success(self, mock_get_connection, valid_insertion_data):
+        '''Tests that valid data doesn't cause an error'''
+        mock_conn = mock.MagicMock()
+        mock_cursor = mock.MagicMock()
+        mock_get_connection.return_value.__enter__.return_value = mock_conn
+        mock_conn.cursor.return_value.__enter__.return_value = mock_cursor
+
+        load_data(valid_insertion_data)
+        mock_conn.commit.assert_called_once()
+
+    @mock.patch('aurora_load.get_connection')
+    def test_load_data_empty_list(self, mock_get_connection):
+        '''Tests that empty data doesn't result in an error'''
+        mock_conn = mock.MagicMock()
+        mock_cursor = mock.MagicMock()
+        mock_get_connection.return_value.__enter__.return_value = mock_conn
+        mock_conn.cursor.return_value.__enter__.return_value = mock_cursor
+
+        load_data(())
+        mock_conn.commit.assert_called_once()
```

Comment:
Nice but it looks like this should have been finished by lunch at least?

Reply:
I agree i agree i agree

Reply:
This guy gets it.